cmd/translate/main_rest_integration_test.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"mime/multipart"
+	"net"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(m *testing.M) {
+	go main()
+
+	// Ensure that a connection can be established.
+	conn, err := net.DialTimeout("tcp", "localhost:8080", time.Second)
+	if err != nil {
+		log.Panic(err)
+	}
+
+	conn.Close()
+
+	os.Exit(m.Run())
+}
+
+func attachFile(text []byte, t *testing.T) (*bytes.Buffer, string) {
+	t.Helper()
+
+	body := &bytes.Buffer{}
+
+	writer := multipart.NewWriter(body)
+	defer writer.Close()
+
+	part, err := writer.CreateFormFile("file", "test.json")
+	if err != nil {
+		log.Panic(err)
+	}
```

Comment:
Inconsistent style with other tests.

```suggestion
	asert.NoError(t, err)
	
	// or
	
	if !assert.NoError(t, err) {
		return nil, ""
	}
```